RiotSwiftUI/Modules/Common/Avatar/ViewModel/AvatarViewModel.swift

```diff
@@ -47,7 +47,7 @@ class AvatarViewModel: InjectableObject, ObservableObject {
         avatarService.avatarImage(mxContentUri: mxContentUri, avatarSize: avatarSize)
             .sink { completion in
                 guard case let .failure(error) = completion else { return }
-                MXLog.error("[AvatarService] Failed to retrieve avatar: \(error)")
+//                MXLog.error("[AvatarService] Failed to retrieve avatar: \(error)")
```

Comment:
It's the only log entry so far in RiotSwiftU so just commented out. In a separate PR I will provide an logger implementation that the app can inject `MXLog` at runtime, removing the compile time dependency on MX.